MessagingService.Testing/TestData.cs

```diff
@@ -38,6 +40,80 @@ public class TestData
 
         public static String ProviderEmailReference = "ProviderEmailReference";
 
+        public static MessageStatus MessageStatusDelivered = MessageStatus.Delivered;
+
+        public static MessageStatus MessageStatusRejected = MessageStatus.Rejected;
+        
+        public static MessageStatus MessageStatusFailed = MessageStatus.Failed;
+        
+        public static MessageStatus MessageStatusBounced = MessageStatus.Bounced;
+        
+        public static MessageStatus MessageStatusSpam = MessageStatus.Spam;
+        
+        public static MessageStatus MessageStatusUnknown = MessageStatus.Unknown;
+
+        public static HttpStatusCode ProviderApiStatusCode = HttpStatusCode.OK;
+
+        public static String ProviderStatusDescription = "delivered";
+
+        public static DateTime DeliveredDateTime = DateTime.Now;
+
+        public static DateTime RejectedDateTime = DateTime.Now;
+
+        public static DateTime BouncedDateTime = DateTime.Now;
+
+        public static DateTime FailedDateTime = DateTime.Now;
```

Comment:
Codacy found an issue: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/manual/stuart_ferguson1/Messaging/pullRequest?prid=5953475)